drivers/storage/portworx/util/util.go

```diff
@@ -148,6 +148,8 @@ const (
 	EnvKeyPortworxAuthJwtIssuer = "PORTWORX_AUTH_JWT_ISSUER"
 	// EnvKeyPortworxAuthSystemAppsKey is an environment variable defining the PX Security shared secret for Portworx Apps
 	EnvKeyPortworxAuthSystemAppsKey = "PORTWORX_AUTH_SYSTEM_APPS_KEY"
+	// EnvKeyAutopilotPXSharedSecret is an environment variable defining the shared secret for Autopilot
+	EnvKeyAutopilotPXSharedSecret = "PX_SHARED_SECRET"
```

Comment:
Can we rename the constant name for stork to something common, instead of having two different variables for stork and autopilot?